src/plugins/numberline/numberline-content.ts

```diff
@@ -0,0 +1,27 @@
+import { types, Instance } from "mobx-state-tree";
+import { TileContentModel } from "../../models/tiles/tile-content";
+import { kNumberlineTileType } from "./numberline-types";
+
+export function defaultNumberlineContent(): NumberlineContentModelType {
+  return NumberlineContentModel.create({text: "Numberline Tile"});
+}
+
+
+export const NumberlineContentModel = TileContentModel
+  .named("NumberlineTool")
+  .props({
+    type: types.optional(types.literal(kNumberlineTileType), kNumberlineTileType),
+    text: "",
```

Comment:
Is this going to be a part of the tile long term? If not, I think it would be best to remove it. If you leave this in now and later remove it, you'll have to accommodate it in old tiles that were made before the change, which sounds like a pain.

Reply:
Got it, I removed `text`